charts/cluster-agent/README.md

```diff
@@ -21,6 +21,15 @@ helm repo update
 
 Kubernetes 1.16+
 
+## Requirements
+
+| Repository | Name | Version |
+|------------|------|---------|
+| https://prometheus-community.github.io/helm-charts/ | metrics-agent(prometheus) | 13.8.0 |
+| https://prometheus-community.github.io/helm-charts/ | mongodb-metrics(prometheus-mongodb-exporter) | 2.8.1 |
+| https://prometheus-community.github.io/helm-charts/ | rabbitmq-metrics(prometheus-rabbitmq-exporter) | 1.0.0 |
+| https://prometheus-community.github.io/helm-charts/ | epsagon-redis-exporter(redis-metrics) | 4.0.2 |
```

Comment:
epsagon-redis-exporter(redis-metrics) | 4.0.2 | i think you have a missmatch between exporter and the name